arthur_bench/models/models.py

```diff
@@ -291,6 +338,7 @@ class TestSuiteSummary(BaseModel):
     total_pages: int
     total_count: int
     num_test_cases: int
+    categorical: bool = False
```

Comment:
do you want to add some type enum here instread of true / false on categorical? unsure whats on the roadmap but if there are other types in the future it may make more sense

Reply:
Switched to output type in the API def but left the `is_categorical` method on the scorer. I think this is slightly better usability for users creating a custom scorer but 🤷‍♀️